tests/unit/src/test/scala/tests/ProcessOutputSuite.scala

```diff
@@ -0,0 +1,57 @@
+package tests
+
+import scala.meta.internal.metals.ProcessOutput
+import scala.collection.mutable
+
+object ProcessOutputSuite extends BaseSuite {
+  def check(
+      name: String,
+      fn: ProcessOutput => Unit,
```

Comment:
`act` is a well established name for those and it also has some meaning ;)

Reply:
Done.